{"name": "Test generate token with invalid accounts", "status": "passed", "parameters": [{"name": "account", "value": "<data_object.account.Account object at 0x00000205A6107770>"}], "start": 1702397949704, "stop": 1702397950843, "uuid": "dd3a4f6a-5634-44c7-a2c5-942c7ae76376", "historyId": "93fd3596c0aa98db5fafd9ccb38039c5", "testCaseId": "573dcf4d17f28e829fe2a39fec07d16d", "fullName": "testcases.api.test_generate_token#test_generate_token_unsuccessfully_with_invalid_account", "labels": [{"name": "parentSuite", "value": "API Test"}, {"name": "suite", "value": "Test Generate Token"}, {"name": "subSuite", "value": "TestGenerateToken"}, {"name": "host", "value": "VNNOT02222"}, {"name": "thread", "value": "18072-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api.test_generate_token"}]}